config/tracing.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"slices"
+)
+
+type TracingConfig struct {
+	Enabled                 bool                 `yaml:"enabled" default:"false"`
+	Attributes              map[string]string    `yaml:"attributes"`
+	Opentelemetry           *OpenTelemetryConfig `yaml:"opentelemetry"`
+	ServiceName             string               `yaml:"service_name" default:"WebhookX"`
+	CapturedRequestHeaders  []string             `yaml:"captured_request_headers"`
+	CapturedResponseHeaders []string             `yaml:"captured_response_headers"`
+	SafeQueryParams         []string             `yaml:"safe_query_params"`
```

Comment:
I don't see the `safe_query_params` is defined in the config.yml? do we really need it?

Reply:
`safe_query_params` defined in config.yml, should check again.